ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java

```diff
@@ -168,7 +171,7 @@ private void processInputUri() throws NullPointerException, IOException {
                 throw e;
             }
         } else if ("content".equals(inputUriScheme)) {
-            String path = FileUtils.getPath(mContext, mInputUri);
+            String path = getFilePath();
```

Comment:
The whole point of this PR is that Storage permission is not mandatory to read image file. 

Fore example, if user selects image from an external app (Gallery etc), then temporary permission is granted for your app to read image file from specific Uri. 
In other words, app has permission to read image file from Uri, but not to query `ContentProvider`.

With null returned we avoid querying `ContentProvider`, instead we try to access the file directly (via `copyFile`).

This way we are able to pick an image from gallery without Storage permission.